DataVisualisation01.pv

```diff
@@ -4,6 +4,8 @@ size(3508, 2480)
 background(1)
 image("correct2.jpg")
 
+showingRegistrationMarks = True
```

Comment:
Addition of this variable `showingRegistrationMarks` makes it easy to toggle on and off the showing of the red dots